recipes/open62541pp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,10 @@
+#include <open62541pp/node.hpp>
+#include <open62541pp/server.hpp>
+
+int main() {
+    int port = 0;
+    opcua::Server server(port);
```

Comment:
As far as I can see, this actually tries to bind a server right? Or is there some kind of initialization needed for that? The examples upstream seem to imply that it does not, and object instantiation enables it by default?

Reply:
But why set port to `0`?

Reply:
Ok, I just wanted to avoid opening any connections, if possible. It was not clear to me that an additional method call is needed for that. I can revert this part.

Reply:
open62541pp's `Server::run` wraps the `UA_Server_run_startup` and `UA_Server_run_iterate` functions.

The `UA_Server_run_startup` will spin-up the event loop and the network layers:

```c
/* Start: Spin up the workers and the network layer and sample the server's
 *        start time.
 * Iterate: Process repeated callbacks and events in the network layer. This
 *          part can be driven from an external main-loop in an event-driven
 *          single-threaded architecture.
 * Stop: Stop workers, finish all callbacks, stop the network layer, clean up */
UA_StatusCode UA_Server_run_startup(UA_Server *server);
UA_UInt16     UA_Server_run_iterate(UA_Server *server, UA_Boolean waitInternal);
UA_StatusCode UA_Server_run_shutdown(UA_Server *server);
```